source/administration/upgrade.rst

```diff
@@ -0,0 +1,232 @@
+Upgrading Mattermost Server
+===========================
+
+In most cases, you can upgrade Mattermost Server in a few minutes, depending on how long it takes for you to back up both the database and the file storage directory.
+
+Which Upgrade Instructions to Use
+---------------------------------
+
+If you are upgrading from version 3.7.0 or greater
```

Comment:
Propose we add a guide for `If you are upgrading from Team Edition to Enterprise Edition` so it's easy for an admin to find it.